README.md

```diff
@@ -32,6 +32,21 @@ public class ExampleApplication extends Application {
 
 **You're good to go!** LeakCanary will automatically show a notification when an activity memory leak is detected in your debug build.
 
+## Releases
```

Comment:
Because it doesn't belong in the CHANGELOG.  An even better solution (in a future PR) is to move stuff out of the README into a wiki or something else.  But for now, it's the Kitchen Sink.

Why worry?  How do other Square projects do it?  Is it better to assume developers can figure out how to reference snapshot builds or provide the minute details?  IMO, Google + StackOverflow are handy resources for filling these gaps, not the project markdown files.